utils/collectd/Makefile

```diff
@@ -501,5 +527,5 @@ $(eval $(call BuildPlugin,vmem,virtual memory usage input,vmem,))
 $(eval $(call BuildPlugin,wireless,wireless status input,wireless,))
 $(eval $(call BuildPlugin,write-graphite,Carbon/Graphite output,write_graphite,+PACKAGE_collectd-mod-write-graphite:libpthread))
 $(eval $(call BuildPlugin,write-http,HTTP POST output,write_http,+PACKAGE_collectd-mod-write-http:libcurl))
-
 $(eval $(call BuildScriptPlugin,sqm,SQM/qdisc collection,sqm_collectd,+PACKAGE_collectd-mod-sqm:collectd-mod-exec))
+$(eval $(call BuildScriptLuaPlugin,ltq-dsl,Lantiq DSL collection,dslstat,+PACKAGE_collectd-mod-ltq-dsl:collectd-mod-lua luci-lib-nixio))
```

Comment:
I think you have missed a `+` before luci-lib-nixio
Also the lua ubus dependency is also missing

Reply:
Are you sure this + is needed? The string is inserted in `(4)` below

```
+  $$(call Package/collectd/Default)                                                                                                                                                                                                                                                                                          
+    TITLE:=$(2) plugin                                                                                                                                                                                                                                                                                                       
+    DEPENDS:= $(4)                                                                                                                                                                                                                                                                                                           
+  endef                                                                                                                                                                                                                                                                                                                      
+         
```

Reply:
I tested on my device and this setup is able to run the script correctly, so it seems there is no need to select anything additional for the ubus import?

Reply:
Please have look on this line https://github.com/openwrt/packages/blob/15f5340a0c43e06e29a25082627c19185a0dccfb/utils/collectd/Makefile#L479 I think we do here the same as in this line

Reply:
Ok changes were applied.